forem/forem
# Creator Onboarding Confirmation Page

**NOTE: This PR can be reviewed in the meantime so that I can incorporate feedback (if any), however in order for the tests to pass, it requires the seed in [PR 14385](https://github.com/forem/forem/pull/14385#pullrequestreview-717928684)**
## What type of PR is this? (check all applicable)


## Description
This PR updates the `confirm-email` view with a new design. 

![page](https://camo.githubusercontent.com/8b118a4cc13bf94cdf97956c508d3c9abda5746a21e13fd46301cc1e42c1d6c0/68747470733a2f2f63646e2d7374642e64726f706c722e6e65742f66696c65732f6163635f3638303736312f664c68595848)

![page](https://camo.githubusercontent.com/d8b3119cfd041972c9526dc8b3f18fba6404fe80138f2525901b4cb247a29150/68747470733a2f2f63646e2d7374642e64726f706c722e6e65742f66696c65732f6163635f3638303736312f6d4736516250)

**What is excluded?**
I've not yet removed the header and footer, but I will in a subsequent design. 

## Related Tickets & Documents
https://github.com/forem/rfcs/pull/190

## QA Instructions, Screenshots, Recordings

1. Add a `creator_onboarding` feature flag to `http://localhost:3000/admin/feature_flags/features`
2. Once its enabled, navigate to `http://localhost:3000/confirm-email?email=ridhwana%40forem.com` and test the page as per the mocks. 

### UI accessibility concerns?

I made sure that its keyboard navigate-able + that voice over works.

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

Does not need to be communicated as yet, will communicate the entire onboarding flow once complete. 
_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
None

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/ARTx27vlCXMxq/giphy.gif)
